crates/sui-graphql-rpc/src/types/display.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
+pub(crate) struct DisplayEntry {
```

Comment:
what is this for?

Reply:
Unrelated to ProtocolConfig -- this is for Object Display Standard support.